logicrunner/goplugin/testplugins/primary/primary.go

```diff
@@ -0,0 +1,34 @@
+package primary
+
+import (
+	"github.com/insolar/insolar/logicrunner"
+	"github.com/insolar/insolar/logicrunner/goplugin/testplugins/foundation"
+)
+
+type INFHwRunner interface {
+	Run() string
+}
+
+// @inscontract
+// nolint
+type HwRunner struct {
+	Runned int
+}
+
+func (h *HwRunner) Run() string {
+	hw := Hw.GetNewInstance(logicrunner.Reference("#1.#2"))
```

Comment:
cannot use logicrunner.Reference("#1.#2") (constant "#1.#2" of type github.com/insolar/insolar/logicrunner.Reference) as Hw value in argument to Hw.GetNewInstance